ocp_resources/model_registry_modelregistry_opendatahub_io.py

```diff
@@ -92,3 +94,23 @@ def to_dict(self) -> None:
                 _spec["postgres"] = self.postgres
 
     # End of generated code
+
+    def create(self, wait: bool = False) -> ResourceInstance | None:
```

Comment:
I don't think this is the place to do it.
do the retry when you call create()

Reply:
`["connection refused", "no endpoints available for service"]` Seems like a valid error, I don't know how the flow should work, but maybe you need to make sure the endpoint is available before this resource?